evm/src/memory/columns.rs

```diff
@@ -4,6 +4,7 @@ use std::ops::Range;
 
 use crate::memory::{NUM_CHANNELS, VALUE_LIMBS};
 
+// Columns for the same memory operations, ordered by (addr, timestamp).
```

Comment:
Nit: "same memory operations" is out of date

Reply:
Good catch